firefox-ios/Client/Frontend/PasswordManagement/PasswordManagerListViewController.swift

```diff
@@ -63,7 +73,14 @@ class PasswordManagerListViewController: SensitiveViewController, Themeable {
     override func viewDidLoad() {
         super.viewDidLoad()
         self.title = .Settings.Passwords.Title
-        tableView.separatorInset = UIEdgeInsets(top: 0, left: 20, bottom: 0, right: 0)
+        let rightInset: CGFloat = {
```

Comment:
Are parentheses necessary? https://www.donnywals.com/swifts-if-and-switch-expressions-explained/